README.adoc

```diff
@@ -113,9 +113,10 @@ Here are the configuration description:
 |kas.policy.create-topic.partition-counter.private-topic-prefix |Specifies the prefix used to match against topic name to detect private/internal topics. |""
 |kas.policy.create-topic.partition-counter.schedule-interval-seconds |Specifies the interval (in seconds) between partition counts. |15
 |kas.policy.create-topic.partition-limit-enforced |`true` enables the enforcement of the partition limit. |false
-|kas.policy.topic-config.enforced |Specifies a mapping between a topic policy key and a value. The property is expressed as a comma separated list of `key:value` pairs where the key and value are separated by  a `:`. Hence the it takes the form `keyA:value1,keyB:value2`. |"flush.ms:9223372036854775807,index.interval.bytes:4096,compression.type:producer,flush.messages:9223372036854775807,segment.jitter.ms:0,min.cleanable.dirty.ratio:0.5,file.delete.delay.ms:60000,segment.index.bytes:10485760,preallocate:false,unclean.leader.election.enable:false"
-|kas.policy.topic-config.mutable |Specifies which properties of topic are allowed to be configured. It's a comma separated list. The configs in `kas.policy.topic-config.enforced` and `kas.policy.topic-config.range config` will be considered as mutable configs by default. |"message.timestamp.difference.max.ms,message.timestamp.type,retention.bytes,min.compaction.lag.ms,cleanup.policy,max.compaction.lag.ms,delete.retention.ms,message.downconversion.enable"
-|kas.policy.topic-config.range |Specifies a mapping from config key to an inclusive/closed range of values. Entries are formatted as a triplet using `:` as a separator, `key:min:max` where either min or max are optional implying a single bound as appropriate. Multiple triplets are comma separated. For example  `configA:minA:maxA,configB:minB:maxB,...`. |"max.message.bytes::1048588,segment.bytes:52428800:,segment.ms:600000:"
+|kas.policy.topic-config.enforced |Specifies a mapping between a topic policy key and a value. The property is expressed as a comma separated list of `key:value` pairs where the key and value are separated by  a `:`. Hence the it takes the form `keyA:value1,keyB:value2`. |[]
+|kas.policy.topic-config.mutable |Specifies which properties of topic are allowed to be configured. It's a comma separated list. The configs in `kas.policy.topic-config.enforced` and `kas.policy.topic-config.range config` will be considered as mutable configs by default. |[]
+|kas.policy.topic-config.range |Specifies a mapping from config key to an inclusive/closed range of values. Entries are formatted as a triplet using `:` as a separator, `key:min:max` where either min or max are optional implying a single bound as appropriate. Multiple triplets are comma separated. For example  `configA:minA:maxA,configB:minB:maxB,...`. |[]
```

Comment:
```suggestion
|kas.policy.topic-config.range |Specifies a mapping from config key to an inclusive/closed range of values. Entries are formatted as a triplet using `:` as a separator, `key:min:max` where either min or max are optional implying a single bound as appropriate. Multiple triplets are comma separated. For example  `configA:minA:maxA,configB::maxB,configC:minC::,...`. |[]
```

I think its worth adding clear(?) examples of all config formats here given there is no default to verify against.

Reply:
Agree!